x-pack/test/alerting_api_integration/security_and_spaces/tests/alerting/alerts.ts

```diff
@@ -533,11 +533,9 @@ instanceStateValue: true
                 savedObjectsClientSuccess: false,
                 callClusterError: {
                   ...searchResult.hits.hits[0]._source.state.callClusterError,
-                  statusCode: 403,
```

Comment:
interesting the 403 is no longer returned like this - I guess it's included in the the `callClusterError` now?  And guessing this is related to the `ignore: [404]` I had a comment on as well?

Reply:
Good catch. The params structure was changed. Will fix it! 👍